app/views/content/edit/_change_notes.html.erb

```diff
@@ -3,7 +3,7 @@
       label: {
         text: t("content.edit.change_note.title"),
       },
-      id: "edition-change-note",
+      id: "change-note-textarea",
```

Comment:
We have a precedent to suffixing these with "field". This precedent was started before field had extra meaning to it but I still think it's ok and preferable to have something less specific.

Reply:
Sure, updated. I think this convention has passed me by!